tts.py

```diff
@@ -0,0 +1,42 @@
+from pathlib import Path
```

Comment:
did we mean to check this in? @smythp

Reply:
It should be in images/nemo/tests. Pushed a second commit to move it but a little too late. Will create a new PR to move it now.